optimum/commands/export/openvino.py

```diff
@@ -167,12 +183,22 @@ def parse_args_openvino(parser: "ArgumentParser"):
             "applying GPTQ takes additional memory and time."
         ),
     )
+    optional_group.add_argument(
+        "--lora",
```

Comment:
I'm afraid this name might be confusing, people can think about efficient fine-tuning, but there's no gradient update and no tuning for a specific task. I'd explicitly name it `lora-correction` and it would be aligned with `nncf` name as well.

Reply:
Thanks for the suggestion! I agree. Changed.